src/main/webapp/i18n/de/modelingEditor.json

```
@@ -3,7 +3,9 @@
         "modelingEditor": {
             "title": "Modellierungseditor",
             "maxScore": "<strong>Max. Punktzahl: {{maxScore}}</strong>",
-            "saveSuccessful": "Dein Diagramm wurde erfolgreich gespeichert!",
+            "allSaved": "Alle Änderungen gespeichert",
+            "unsavedChanges": "Nicht gespeichert",
+            "saving": "Speichern ...",
```

```suggestion
            "saving": "Speichern…",
```
Nitpick: there shouldn’t be a space before the ellipsis.